src/test/javascript/spec/component/lecture/lecture.component.spec.ts

```diff
@@ -228,4 +233,14 @@ describe('Lecture', () => {
         expect(lectureComponent.filteredLectures).toBeArrayOfSize(1);
         expect(lectureComponent.filteredLectures).toContainEqual(unspecifiedLecture);
     });
+
+    it('should sort rows', () => {
```

Comment:
This test simply checks if the `sortyByProperty` function is called. It would be better if you could include more than 1 element in `filteredLectures` and check if the result of sorting is actually correct.

Reply:
Thanks for the suggestion! I've added more elements to `filteredLectures` and will now verify if the sorting is accurate.